app/portainer/views/endpoints/edit/endpointController.js

```diff
@@ -265,18 +267,37 @@ function EndpointController(
         if (endpoint.Type === PortainerEndpointTypes.EdgeAgentOnDockerEnvironment || endpoint.Type === PortainerEndpointTypes.EdgeAgentOnKubernetesEnvironment) {
           $scope.edgeKeyDetails = decodeEdgeKey(endpoint.EdgeKey);
           $scope.randomEdgeID = uuidv4();
-
           $scope.state.availableEdgeAgentCheckinOptions[0].key += ` (${settings.EdgeAgentCheckinInterval} seconds)`;
         }
 
         $scope.endpoint = endpoint;
         $scope.groups = groups;
         $scope.availableTags = tags;
 
+        if (settings.FeatureFlagSettings && settings.FeatureFlagSettings['open-amt']) {
+          openAmt = true;
+        }
+
         configureState();
       } catch (err) {
         Notifications.error('Failure', err, 'Unable to retrieve environment details');
       }
+
+      if (openAmt) {
```

Comment:
why not make the `openAmt` declaration here and in a single line?
```suggestion
      const showAMTInfo = settings.FeatureFlagSettings && settings.FeatureFlagSettings['open-amt']
      if (showAMTInfo) {
```

Reply:
no. that bool is set from the previous async that has access to the settings. separating this is how I'm preventing something going wrong with the amt API from breaking the rendering of the non-amt config.

I also suspect you'll be moving this code to a different UI, so its not worth guilding.